infrastructure/iac/types.go

```diff
@@ -40,3 +40,17 @@ type iacIssue struct {
 	Documentation  lsp.Uri        `json:"documentation"`
 	IacDescription iacDescription `json:"iacDescription"`
 }
+
+type IssueData struct {
```

Comment:
The copy of this type should go to lsp types declaration (`message_types.go`) as well. You can do it in this PR, or later when you report IaC issues, up to you.